src/utility/firebase.js

```diff
@@ -37,3 +38,29 @@ export const getSponsors = () => {
       return querySnapshot.docs
     })
 }
+
+export const getProject = async (user_id, setProjectCallback, setFeedbackCallback) => {
```

Comment:
I was thinking that the Judging page (which displays 5 projects) will need to use something similar to `getProject` but doesn need a feedback callback. Should I just call this function and pass an empty `setFeedbackCallback` or should we split this out to two functions?

Reply:
i'm thinking keep the `setFeedbackCallback` empty and we can check for empty callback in the `getProject` function and not execute that part of the code, will keep it cleaner